ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -246,96 +239,78 @@ export default class UnlockPage extends Component {
-                  <Text variant={TextVariant.bodyMdMedium}>
-                    {t('password')}
-                  </Text>
-                </Box>
-              }
-              placeholder={t('enterPassword')}
+              placeholder={t('enterYourPassword')}
               size={FormTextFieldSize.Lg}
               inputProps={{
                 'data-testid': 'unlock-password',
-                type: InputType.Password,
+                'aria-label': t('password'),
               }}
               onChange={(event) => this.handleInputChange(event)}
+              type={TextFieldType.Password}
+              value={password}
               error={Boolean(error)}
               helpText={this.renderHelpText()}
               autoComplete
               autoFocus
               disabled={isLocked}
               width={BlockSize.Full}
-              textFieldProps={{
-                borderRadius: BorderRadius.LG,
-              }}
+              marginTop={4}
+              marginBottom={4}
             />
-          </Box>
-          <Box
-            display={Display.Flex}
-            flexDirection={FlexDirection.Column}
-            width={BlockSize.Full}
-            alignItems={AlignItems.center}
-          >
-            <Box
-              className="unlock-page__buttons"
-              display={Display.Flex}
-              flexDirection={FlexDirection.Column}
-              width={BlockSize.Full}
-              gap={4}
+            <Button
+              variant={ButtonVariant.Primary}
+              size={ButtonSize.Lg}
+              block
+              type="submit"
+              data-testid="unlock-submit"
+              disabled={!password || isLocked}
+              marginBottom={6}
             >
-              <Button
-                variant={ButtonVariant.Primary}
-                size={ButtonSize.Lg}
-                block
-                type="submit"
-                data-testid="unlock-submit"
-                disabled={!password || isLocked}
-              >
-                {this.context.t('unlock')}
-              </Button>
-              <ButtonLink
-                data-testid="unlock-forgot-password-button"
-                key="import-account"
-                type="button"
-                onClick={() => this.onForgotPassword()}
-              >
-                {t('forgotPassword')}
-              </ButtonLink>
-            </Box>
-            <Box marginTop={6} className="unlock-page__support">
+              {this.context.t('unlock')}
+            </Button>
+
+            <Button
+              variant={ButtonVariant.Link}
+              data-testid="unlock-forgot-password-button"
+              onClick={() => this.onForgotPassword()}
+              marginBottom={6}
+            >
+              {t('forgotPassword')}
+            </Button>
+
+            <Text marginBottom={8}>
               {t('needHelp', [
-                <a
+                <Button
+                  variant={ButtonVariant.Link}
```

Comment:
Replacing raw html for design system component. Will still render as an <a> tag, check snapshot